src/main/java/org/symphonyoss/symphony/messageml/elements/Element.java

```diff
@@ -770,6 +782,23 @@ void assertChildrenNotExceedingMaxCount(Collection<Class<? extends Element>> ele
     }
   }
 
+  /**
+   * Given a target id present in a UIAction this method returns the corresponding dialog associated to the same
+   * id, if no dialog is found it throws an exception
+   */
+  Dialog checkMatchingDialog(String targetId) throws InvalidInputException {
```

Comment:
checkMatching -> findMachingDialog? (as we don't really check the dialog itself but just look for one)